ops/_tracing/__init__.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The tracing facility of the Operator Framework.
+
+TODO: quick start, usage example.
+"""
+
+from __future__ import annotations
+
+from contextlib import contextmanager
+from typing import Generator
+
+import opentelemetry.trace
+
+import ops.version
+
+tracer = opentelemetry.trace.get_tracer('ops', ops.version.version)
```

Comment:
I think the expected way to do this is:

```suggestion
import ops

tracer = opentelemetry.trace.get_tracer('ops', ops.__version__)
```

Reply:
I used `ops.version.version` to avoid circular imports.
As this `tracer` is reused in other modules, it has to be created early, and since this is a submodule, I don't think `import ops` or `from ops import __version__` is possible.

Please teach me if you know a way!

Reply:
no longer relevant, redone.